api/backend/infrastructure/repositories/ingestion.py

```
@@ -0,0 +1,422 @@
+        file_path: str,
+        auth_file_path: str,
+        required_road_types: tuple[str, ...],
+        city_name: str,
+    ) -> None:
+        """Filter the original PBF and load the result into PostgreSQL tables."""
+        # Sanitize city_name to prevent path traversal
+        import re
+
+        sanitized_name = re.sub(r"[^\w\-]", "_", city_name)
+        if sanitized_name != city_name:
+            raise ValueError("Invalid city_name: contains unsafe characters")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)